### DISCUSSION TITLE
mac 10.15.7 amd二进制安装成功之后，启动时一直报错

### DISCUSSION THREAD

**Discussion by tiantsl** [clickvisual/clickvisual#919] [Q&A] [answered]
mac 10.15.7 amd二进制安装成功之后，启动时一直报错，配置文件中，MySQL和redis配置已做了修改，还是仍然报错。报错如下:
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/38754898/216547392-5bf29816-f082-43b0-a99a-8ccc7fb0d202.png">

**Comment by kl7sn** [comment, marked as answer]
该问题已解决，关联 issues: https://github.com/clickvisual/clickvisual/issues/920
